### PR TITLE
Fixed openjdk download when jdk_version is next

### PR DESCRIPTION
**Pull request by LongyuZhang**
- When jdk_version is next, download repo will be openjdk-jdk or openj9-openjdk-jdk.
[ci skip]
Issue: #1176

Signed-off-by: Longyu Zhang <longyu.zhang@ibm.com>

### CODE REVIEW

**Comment by LongyuZhang** [comment]
reviewers: @llxia @renfeiw

File: buildenv/jenkins/JenkinsfileBase [hunk 1]
```diff
@@ -41,9 +41,11 @@ def setupEnv() {
 	}
 
 	if ( params.JDK_VERSION ) {
+		env.ORIGIN_JDK_VERSION = params.JDK_VERSION
 		env.JDK_VERSION = params.JDK_VERSION.equalsIgnoreCase("next") ? "" : params.JDK_VERSION
 	} else if ( params.JAVA_VERSION ) { // temporarily support JAVA_VERSION
 		env.JDK_VERSION = (params.JAVA_VERSION.trim())[2..-2];
+		env.ORIGIN_JDK_VERSION = (params.JAVA_VERSION.trim())[2..-2];
```
Comment (llxia) [inline buildenv/jenkins/JenkinsfileBase]: This is not needed. We are going to remove `JAVA_VERSION` soon.

**Comment by llxia** [comment]
Please link to the issue and provide Grinder links. Thanks.

**Comment by LongyuZhang** [comment]
This PR has been linked with the issue, and following are the Grinder links, thanks.
Link for openj9 with jdk_version next: https://ci.adoptopenjdk.net/view/work%20in%20progress/job/Grinder_Sandbox/564/
Link for openj9 with jdk_version blank: https://ci.adoptopenjdk.net/view/work%20in%20progress/job/Grinder_Sandbox/566/
Link for hotspot with jdk_version next: https://ci.adoptopenjdk.net/view/work%20in%20progress/job/Grinder_Sandbox/563/
Link for hotspot with jdk_version blank: https://ci.adoptopenjdk.net/view/work%20in%20progress/job/Grinder_Sandbox/565/

**Pull request merged**